config.example.wikimedia.yaml

```diff
@@ -40,6 +40,9 @@ default_project: &default_project
           cache_control: s-maxage=600, max-age=600
           uri: https://pdf-electron.wmflabs.org
           secret: secret
+        trending:
+          host: https://trending-beta.wmflabs.org
+          cache-control: s-maxage=600, max-age=600
```

Comment:
Let's start conservatively with 30 or 45 mins and then decrease it once we see how the service behaves in production.